### PR TITLE
In AntiSpoof connect to UserRename::AfterGlobal hook

### PR DESCRIPTION
**Pull request by sqreek**
https://wikia-inc.atlassian.net/browse/SUS-882

@mixth-sense

### CODE REVIEW

**Comment by Grunny** [comment]
Does this still block spoofing the old username in addition to the new name? The comment on the method says "On rename, remove the old entry and add the new", so it suggests it does not.

Unlike the MW core rename extension, our user rename tool creates the old user name so it can't be taken, and the old names probably still shouldn't be spoofable (especially for users who were well known under their previous username). The MW core rename tool doesn't create the old name, as renaming frees the old name for reuse by someone else, so it wasn't an issue for them.

**Comment by sqreek** [comment]
That is true, it's for renaming, so old names will be available after this change.
Let me fix that.

**Comment by Kirkburn** [comment]
Though you're already tweaking it: I agree that old usernames should remain covered by antispoof. We wouldn't want a newly renamed user to potentially have their old name start being impersonated around the site.

File: extensions/AntiSpoof/AntiSpoofHooks.php [hunk 1]
```diff
@@ -123,4 +123,22 @@ public static function asAddRenameUserHook( $uid, $oldName, $newName ) {
...
+	 * (After a successful user rename using Wikia Tool)
+	 *
+	 * @param $dbw DatabaseBase
+	 * @param $uid
+	 * @param $oldusername string
+	 * @param $newusername string
+	 * @param $process
+	 * @param $tasks
+	 */
+
```
Comment (Grunny) [inline extensions/AntiSpoof/AntiSpoofHooks.php]: Minor: remove this blank line between the comment and the method.

**Comment by Grunny** [comment]
LGTM.

File: extensions/AntiSpoof/AntiSpoofHooks.php (hunk 1 above)
Comment (sqreek) [inline extensions/AntiSpoof/AntiSpoofHooks.php]: removed, and rebased

**Comment by sqreek** [comment]
Closing in favor of #11131

**Pull request closed without merging**